platform-ui-skin/src/main/webapp/skin/less/social/portlets/multifactorAuthentication/Style.less

```diff
@@ -0,0 +1,202 @@
+  }
+  .switch {
+    position: relative;
+    display: inline-block;
+    width: 53px;
+    height: 32px;
+    top: 0.4rem;
+  }
+
+  .switch input {
+    display: none;
+  }
+
+  .slider {
+    position: absolute;
+    cursor: pointer;
+    overflow: hidden;
+    top: 5px;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    width: 60px;
+    height: 20px;
+    background-color: @lightGrey;
+    -webkit-transition: .4s;
+    transition: .4s;
+  }
+
+  .slider:before {
+    position: absolute;
+    z-index: 2;
+    content: "";
+    height: 14px;
+    width: 14px;
+    left: 5px;
+    bottom: 3px;
+    background-color: @darkGrey;
+    -webkit-box-shadow: 0 2px 5px rgba(0, 0, 0, 0.22);
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.22);
+    -webkit-transition: .4s;
+    transition: all 0.4s ease-in-out;
+  }
+
+  input:not(:checked)+.slider > .absolute-activate {
+    z-index: -1;
+    position: absolute;
+    -webkit-transform: translateX(-190px);
+    -ms-transform: translateX(-190px);
+    transform: translateX(-190px);
+    left: 2px;
+    color: @white;
+    text-align: right;
+    font-size: 9px;
+    width: calc(100% - 40px);
+    line-height: 22px;
+    cursor: pointer;
+    transition: all 0.4s ease-in-out;
+  }
+
+  input:checked+.slider> .absolute-activate {
+    -webkit-transform: translateX(0px);
+    -ms-transform: translateX(0px);
+    transform: translateX(0px);
+    z-index: 15;
+    position: absolute;
+    left: 2px;
+    color: @white;
+    text-align: right;
+    font-size: 13px;
+    width: calc(100% - 40px);
+    line-height: 22px;
+    cursor: pointer;
+    transition: all 0.4s ease-in-out;
+  }
+
+  .slider:after {
+    position: absolute;
+    left: -20px;
+    z-index: 1;
+    content: "";
+    font-size: 13px;
+    text-align: left !important;
+    line-height: 19px;
+    padding-left: 0;
+    width: 95px;
+    height: 26px !important;
+    color: @tableBackgroundAccent;
+    background-color: #477ab3;
+    background-image: -moz-linear-gradient(top, @primaryColorDefault, #2f5e92);
+    background-image: -webkit-gradient(linear, 0 0, 0 100%, from(#578dc9), to(#2f5e92));
+    background-image: -webkit-linear-gradient(top, @primaryColorDefault, #2f5e92);
+    background-image: -o-linear-gradient(top, @primaryColorDefault, #2f5e92);
+    background-image: linear-gradient(to bottom, @primaryColorDefault, #2f5e92);
+    background-repeat: repeat-x;
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#ff578dc9', endColorstr='#ff2f5e92', GradientType=0);
+    -webkit-box-shadow: inset 0px 3px 5px @primaryDarkerColor;
+    -moz-box-shadow: inset 0px 3px 5px @primaryDarkerColor;
+    box-shadow: inset 0px 3px 5px @primaryDarkerColor;
+    -webkit-border-top-left-radius: 9px;
+    -moz-border-radius-topleft: 9px;
+    border-top-left-radius: 9px;
+    -webkit-border-bottom-left-radius: 9px;
+    -moz-border-radius-bottomleft: 9px;
+    border-bottom-left-radius: 9px;
+    height: 57px;
+    border-radius: 100px;
```

Comment:
if you define border-radius here, it overwrite border-top-left-radius and border-bottom-left-radius
So, one of theses is not needed